### PR TITLE
add support for wikipedia, readthedocs, cnblogs, etc. ; add keyboard shortcut: alt+t

### PR DESCRIPTION
**Pull request by davuses**
add support for wikipedia, readthedocs, cnblogs, etc. ; add keyboard shortcut for toggling: alt+t

### CODE REVIEW

File: toc-bar.user.js [hunk 1]
```diff
@@ -628,6 +694,14 @@ a.toc-link {
         this.toggle()
         GM_setValue('tocbar-hidden', !this.visible)
       })
+      // toggle keyboard shortcut: alt+t
+      let tocBar = this;
+      document.addEventListener('keydown', function(e) {
```
Comment (hikerpig) [inline toc-bar.user.js]: I'm afraid this is too intrusive for the user.
Can we use some of GM or violent monkey's shortcut feature (if there is any) so the user can see and change this shortcut ?

File: toc-bar.user.js [hunk 2]
```diff
@@ -446,11 +509,14 @@
   list-style: none;
   padding-left: 8px;
   position: static;
+  margin: auto;
```
Comment (hikerpig) [inline toc-bar.user.js]: What does this do?

File: toc-bar.user.js [hunk 3]
```diff
@@ -310,6 +371,7 @@
 
   position: fixed;
   z-index: 9000;
+  font-family: 'Roboto';
```
Comment (hikerpig) [inline toc-bar.user.js]: I'm not sure if this should be added, or maybe it's not enough to add just one font-family

File: toc-bar.user.js [hunk 4]
```diff
@@ -446,11 +509,14 @@
...
   padding-left: 8px;
   position: static;
+  margin: auto;
+  font-size: 1em;
 }
 
 a.toc-link {
   color: currentColor;
   height: 100%;
+  text-decoration: auto;
```
Comment (hikerpig) [inline toc-bar.user.js]: What will happen if we don't add this ?

**Review by hikerpig — CHANGES_REQUESTED** [review]
Thank you for this PR !
But I have some little questions, followed by the comments.

**Comment by davuses** [comment]
Currently tocbar might look differently on different sites :
 image
image
image

**Review by hikerpig — APPROVED** [review]
Appreciate your efforts 🙇‍♀️

**Pull request merged**